src/components/searchResults/UserContact.js

```diff
@@ -0,0 +1,39 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import "./UserContact.css";
+
+const UserContact = (props) => {
+  return (
+    <div className="user-contact">
+      <FontAwesomeIcon
+        icon={props.icons[0]}
+        className="icons grid-item grid-item__1"
+      />
+      <p className="icon-text grid-item grid-item__2">
+        {props.contactInfo.location}
+      </p>
+      <FontAwesomeIcon
+        icon={props.icons[1]}
+        className="icons grid-item grid-item__3"
+      />
+      <p className="icon-text grid-item grid-item__4">
+        {props.contactInfo.twitter}
+      </p>
+      <FontAwesomeIcon
+        icon={props.icons[2]}
+        className="icons grid-item grid-item__5"
+      />
+      <a href="#" target="_blank" className="link">
+        {props.contactInfo.website}
+      </a>
+      <FontAwesomeIcon
+        icon={props.icons[3]}
+        className="icons grid-item grid-item__7"
+      />
+      <p className="icon-text grid-item grid-item__8">
+        @{props.contactInfo.job}
+      </p>
+    </div>
+  );
+};
+
```

Comment:
1. These icons and texts should be links, because they represent Twitter profiles, blogs, Github Urls and locations.

2. A way to reduce the code on the file is to have the data in an array (passed down from a parent component) then loop through them here - it would look something like this:

```suggestion
const UserContact = (props) => {
  const iconData = [
    { name: "Lagos", icon: "locationicon" },
    { name: "@me", icon: "twittericon" },
    { name: "email@me", icon: "emailicon" },
  ];
  return (
    <div className="user-contact">
      {iconData.map((data) => (
        <a href={`www.test.com/${data.name}`}>
          {data.name}
          <FontAwesomeIcon icon={data.icon} className="icon" />
        </a>
      ))}
    </div>
  );
};
```

Reply:
Okay. I understand this also and have effected the change.. 
Is test.com just a dead link like #? Seems so, but I can't be sure.